### ISSUE TITLE
[DeepLearning] Homework 2 

### ISSUE THREAD

**Issue by eunbin777** [eunbin777/MachineLearning#7]
2.1 Basic CNN model
- Generate two classification models for FMNIST/CIFAR10 datasets with keras sequential/functional API
- with your own CNN model (# of convolutional layers, filters, activation function, optimizer, …)
- Print train/validation loss of your own model

2.2 ResNet
- Generate a transfer model for CIFAR10 dataset with pretrained ResNet50 model
- with your own freezing point if possible (refer to https://keras.io/guides/transfer_learning/)
- with your own fully connected layers (# of layers, nodes, activation function, …)
- Print train/validation loss of your own models
